src/utils/video-listing.jsx

```diff
@@ -0,0 +1,5 @@
+export const getFilteredVideos = ([...videos], state) => {
```

Comment:
Argument named state, has different descriptive meaning than for what being used here.